packages/eslint-plugin/src/rules/consistent-type-exports.ts

```diff
@@ -193,46 +193,20 @@ export default createRule<Options, MessageIds>({
             const allExportNames = report.typeBasedSpecifiers.map(
               specifier => specifier.local.name,
             );
-
-            if (allExportNames.length === 1) {
-              const exportNames = allExportNames[0];
```

Comment:
This line was lost in the transition.

Reply:
this was actually not needed, because `formatWordList` is able to handle a `1`-length array